api/v1/qdrantcluster_types.go

```diff
@@ -162,6 +171,10 @@ type QdrantClusterSpec struct {
 	// to inspect the attached storage.
 	// +optional
 	StartupDelaySeconds *int `json:"startupDelaySeconds,omitempty"`
+	// RebalanceStrategy specifies the strategy to use for automaticially rebalancing shards the cluster.
+	// Cluster-manager needs to be enabled for this feature to work.
+	// +optional
```

Comment:
if it is optional, what is then the default? could we set it here?

Reply:
The default is that no rebalance strategy is set.